Makefile

```diff
@@ -47,4 +47,7 @@ REALDEPDIR = $(shell realpath $(DEPDIR))
 
 .PHONY: test
 test:
-	LD_LIBRARY_PATH=$(REALDEPDIR) go test -v -ldflags "-extldflags \"-L$(REALDEPDIR) -led25519_bip32\"" ./...
+	CGO_CFLAGS="-I$(REALDEPDIR)" \
+	CGO_LDFLAGS="-L$(REALDEPDIR)" \
+	LD_LIBRARY_PATH=$(REALDEPDIR) \
+	go test -v -count 1 -ldflags "-extldflags \"-L$(REALDEPDIR) -led25519_bip32 -lspacemesh_remote_wallet\"" ./...
```

Comment:
TIL that `-count 1` can be used to disable caching for tests :+1: I always used `go clean -testcache` before 🤦